tests/create_LWE.py

```diff
@@ -0,0 +1,231 @@
+    mct.axial_dispersion = 5.75e-8
+
+    configure_discretization(mct, options.n_col, options.n_partype, options.n_par, options.n_rad, options.ad_jacobian)
+    configure_particles(mct, options.n_partype)
+    configure_adsorption(mct, component_system, options.binding_model, options.is_kinetic)
+    configure_film_diffusion(mct, options.n_comp, options.n_partype)
+    configure_pore_diffusion(mct, options.n_comp, options.n_partype)
+    configure_flow_direction(mct, options.reverse_flow)
+
+def configure_discretization(unit_operation, n_col, n_par, ad_jacobian):
+
+    unit_operation.discretization.ncol = n_col
+    unit_operation.discretization.npar = n_par
+    unit_operation.discretization.par_disc_type = 'EQUIDISTANT_PAR'
+    unit_operation.discretization.use_analytic_jacobian = not ad_jacobian
+    unit_operation.discretization.max_krylov = 0
+    unit_operation.discretization.gs_type = True
+    unit_operation.discretization.max_restarts = 10
+    unit_operation.discretization.schur_safety = 1e-8
+    unit_operation.discretization.weno = {'boundary_model': 0, 'weno_eps': 1e-10, 'weno_order': 3}
+
+def configure_particles(unit_operation):
+    par_radius = 4.5e-5
+    par_porosity = 0.75
+
+    unit_operation.particle_radius = par_radius
+    unit_operation.particle_porosity = par_porosity
+
+    unit_operation.discretization.par_geom = 'SPHERE'
+ 
+
+def get_binding_model_class(binding_type, component_system):
+
+    binding_classes = {
+    'linear': Linear,
+    'langmuir': Langmuir,
+    'langmuir_ldf': LangmuirLDF,
+    'bilangmuir': BiLangmuir,
+    'bilangmuir_ldf': BiLangmuirLDF,
+    'freundlich_ldf': FreundlichLDF,
+    'steric_mass_action': StericMassAction,
+    'anti_langmuir': AntiLangmuir,
+    'spreading': Spreading,
+    'self_association': SelfAssociation,
+    'bi_steric_mass_action': BiStericMassAction,
+    'multistate_steric_mass_action': MultistateStericMassAction,
+    'simplified_multistate_steric_mass_action': SimplifiedMultistateStericMassAction,
+    'saska': Saska,
+    'generalized_ion_exchange': GeneralizedIonExchange
+    }
+
+    try:
+        binding_class = binding_classes[binding_type]
+        return binding_class(component_system)
+    except KeyError:
+        raise ValueError(f'Unknown binding type {binding_type}')
+
+
+def configure_adsorption(unit_operation, component_system, binding_type, is_kinetic):
+    
+    kA = 35.5
+    kD = 1000.0
+    nu = 4.7
+    sigma = 11.83
+    sma_lambda = 1.2e3
+
+    binding_type = 'steric_mass_action'
+    binding_model = get_binding_model_class(binding_type, component_system)
+
+    binding_model.is_kinetic = is_kinetic
+
+    unit_operation.binding_model = binding_model
+    unit_operation.n_bound_states = [1] * 4 #n_comp
+    unit_operation.adsorption_rate = kA
+    unit_operation.desorption_rate = kD
+    unit_operation.characteristic_charge = nu
+    unit_operation.steric_factor = sigma
+    unit_operation.capacity = sma_lambda
+
```

Comment:
[formatting] I would encourage you to have some form of linter that helps with formatting the code according to [PEP 8](https://peps.python.org/pep-0008/#blank-lines). 

E.g. 
> Surround top-level function and class definitions with two blank lines.